packages/react-server/core/context/Navigator.js

```diff
@@ -215,11 +220,7 @@ class Navigator extends EventEmitter {
 			}
 
 			this.emit('navigateDone', null, page, request.getUrl(), type);
-		}).catch(err => {
-			logger.error("Error while handling route", err);
-
-			this.emit('navigateDone', {status: 500}, page, request.getUrl(), type);
-		});
+		}).catch(err => this.emit('navigateDone', err, page, request.getUrl(), type));
```

Comment:
That's a good question- I think one could make the argument either way. Let's discuss further this afternoon.